recipes/winmd/all/conanfile.py

```diff
@@ -0,0 +1,70 @@
+import os
+
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import copy, get
+from conan.tools.layout import basic_layout
+from conan.tools.scm import Version
+from conan.tools.microsoft import is_msvc, check_min_vs
+
+required_conan_version = ">=1.52.0"
+
+
+class WinMDConan(ConanFile):
+    name = "winmd"
+    description = "C++ winmd parser"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/microsoft/winmd"
+    topics = ("native", "C++", "WinRT", "WinMD")
```

Comment:
```suggestion
    topics = ("native", "cpp", "winrt", "winmd")
```
Prefer lower-case always, and avoid symbols. It should consider search engine when using topics.